ui-components/src/locales/bn.json

```diff
@@ -1356,7 +1356,26 @@
         "noResultsSubtitle": "সেরা ফলাফলের জন্য ফিল্টার সম্পাদনা বা অপসারণ করুন।",
         "includeUnknowns": "অসম্পূর্ণ তথ্য সহ ফলাফল অন্তর্ভুক্ত করুন",
         "senior": "সিনিয়র হাউজিং (62+)",
-        "specialNeeds": "বিশেষ চাহিদা এবং অক্ষমতা"
+        "specialNeeds": "বিশেষ চাহিদা এবং অক্ষমতা",
+        "minAmiPercentageLabel": "যাদের আয় আছে তাদের জন্য ইউনিট আলাদা রাখা হয়েছে",
+        "minAmiPercentageOptions": {
+          "amiOption20": "এএমআই এর 20% বা তার বেশি",
+          "amiOption25": "25% AMI বা তার বেশি",
+          "amiOption30": "এএমআই এর 30% বা তার বেশি",
```

Comment:
Just curious if you know why some of these have `25% AMI` and others don't actually mention it?

Reply:
I just copied from google translate so I have no idea.